src/Sarif.Driver/Sdk/ExportRulesDocumentationCommandBase.cs

```diff
@@ -46,13 +48,20 @@ internal void BuildRule(Skimmer<TContext> rule, StringBuilder sb)
         {
             sb.AppendLine($"## Rule `{rule.Moniker}`{Environment.NewLine}");
             sb.AppendLine($"### Description{Environment.NewLine}");
-            sb.AppendLine($"{rule.FullDescription.Text}{Environment.NewLine}");
+            sb.AppendLine($"{rule.FullDescription.Text ?? rule.ShortDescription.Text ?? "No description available."}{Environment.NewLine}");
```

Comment:
>"No description available." [](start = 88, length = 27)

Resource string `NoRuleDescription`. Your message text is fine.

Reply:
Should be a resource string with that resource identifier (not a local const string).

---
In reply to: [493028111](https://github.com/microsoft/sarif-sdk/pull/2082#discussion_r493028111) [](ancestors = 493028111)